### ISSUE TITLE
Some changes rolled back on IcingaDB support?

### ISSUE THREAD

**Issue by Tontonitch** [HenriWahl/Nagstamon#1006]
Seems that https://github.com/HenriWahl/Nagstamon/pull/985 and https://github.com/HenriWahl/Nagstamon/pull/982 had been rolled back. I don't see since when, but the merged changes are not in master anymore. So both related problems are back in latest build.

I've forked again your repo, put back my changes and built a release for windows (I cannot test on other systems).

We are now using that new release in production without problems, connecting to Icinga2/IcingaDBWeb systems with IcingaDB 1.0.2 and 1.1.1. So the compatibility is fine for IcingaDB < or >= 1.1 and it timestamp/date format change.
(this change is needed to be able to proceed with a smooth upgrade of our monitoring servers)

Concerning the changes I've done, to fix the timezone problem I had to import python lib "timezone", and did it in the way "from datetime import timezone" : 
https://github.com/HenriWahl/Nagstamon/commit/3e5a3ba70a7a760cdb8687003c2e5d68c7f3e4e3
On my windows builds it works, on other systems I don't know if it is the case. So please tell me if this change is fine or if I need to adapt something.

**Comment by HenriWahl** [comment]
Hi @Tontonitch if so this happened by accident. I am going to investigate.

**Comment by big-elno** [comment]
Hi @HenriWahl 
I can confirm that the issue #982 is fixed again in latest stable.

**Comment by HenriWahl** [comment]
@big-elno @Tontonitch so this issue could be closed?

**Comment by Tontonitch** [comment]
Hello @HenriWahl,
I've checked the code and all the changes are indeed back again, thanks.
I did not try the release from your master branch, as I had to go ahead with a migration of icingadb on my side, so needed to switch to a working version, so I took the release from my fork. We work with it since 2-3 weeks without any issue, so I assume the version from your master branch is fine too. I will switch back to your version as soon as your next release will go out.
